frontend/simple/views/components/i18n.vue

```diff
@@ -1,11 +1,37 @@
 <template>
-    <span><slot></slot></span>
+    <component :is="tag" v-on="$listeners">
+      {{translatedText}}
+    </component>
 </template>
 <script>
-export default{
-  props: {args: [Object, Array]},
-  beforeMount: function () {
-    this.$slots.default[0].text = this.L(`${this.$slots.default[0].text}.text`, this.args || [], {defaultValue: this.$slots.default[0].text})
+export default {
+  props: {
+    args: [Object, Array],
+    tag: {
+      type: String,
+      default: 'span'
+    }
+  },
+  computed: {
+    translatedText () {
+      const text = this.L(
+        this.$slots.default[0].text,
+        this.args || {},
+        { defaultValue: this.$slots.default[0].text }
+      )
+
+      // BUG: Sometimes text is not returned correctly
+      // Could not understand why. You can see it happen on
+      // 'Pay Group' link on side bar and at /pay-group page - Title: "Pay Group"
+      // When that happens, just return the default text.
```

Comment:
I've fixed this, I will post comments to the PR with instructions on how to fix it.